config.go

```diff
@@ -1,11 +1,12 @@
 package main
 
 import (
+	"strings"
```

Comment:
Pointless change

Reply:
Don't know why that happened, will remove

Reply:
Hm, I can't seem to save my file without it appearing at the top. Perhaps it's something to do with a VS Code extension I have installed - doing local imports before Github imports... either way, I'll remove the change